README.md

```diff
@@ -74,30 +74,164 @@ An example of one such record:
 }
 ```
 
+
 ## Making Queries
 
+
 All queries to the index are made through HTTP using JSON data payloads.
 This gives a simple means of interaction that is easily accessible to any
 number of languages.
 
 These queries are handled via requests and wrapped into the index client.
 
-### Create a record
 
-***TODO***
+  ### Create a record ###
+
+
+# Method: create
+
+Required Arguments:
```

Comment:
i actually would not include any of this or optional args since the code may be changed and we shouldn't have to update this too. So honestly I'd just have the `Example` without this.